### PR TITLE
8). Fix toc.yml and topic files

### PR DESCRIPTION
**Pull request by jameshkramer**
@mairaw
This PR is for user story #1576037.

It fixes toc.yml (extensively, because of How to: -> How to) and makes minor changes to topic files.

cc: @tfosmark

### CODE REVIEW

File: docs/standard/linq/overview-linq-xml.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ dev_langs:
 ms.assetid: 716b94d3-0091-4de1-8e05-41bc069fa9dd
 ---
 
-# LINQ to XML overview
+# Overview (LINQ to XML)
```
Comment (mairaw) [inline docs/standard/linq/overview-linq-xml.md]: I think I'd keep the current filename and title

File: docs/standard/linq/toc.yml [hunk 2]
```diff
@@ -302,21 +302,21 @@
           href: program-nodes.md
         - name: Mixed declarative code-imperative code bugs
           href: mixed-declarative-code-imperative-code-bugs.md
-        - name: "How to: Stream XML fragments with access to header information"
+        - name: "How to stream XML fragments with access to header information"
           href: stream-xml-fragments-access-header-information.md
-        - name: "How to: Perform streaming transform of large XML documents"
-          href: perform-streaming-transform-large-xml-documents.md
-        - name: "How to: Read and write an encoded document"
+        - name: "How to do streaming transform of large XML documents"
```
Comment (mairaw) [inline docs/standard/linq/toc.yml]: I know that Acrolinx complains about perform but I'd keep it here

**Review by mairaw — CHANGES_REQUESTED** [review]
Changes look good @jameshkramer but left a few comments for you to consider.

File: docs/standard/linq/overview-linq-xml.md (hunk 1 above)
Comment (jameshkramer) [reply docs/standard/linq/overview-linq-xml.md]: I did the filename and title to be consistent with the title: Classes overview - LINQ to XML

In other topics that have - LINQ to XML in the title, we put (LINQ to XML) in the H1, and we don't put anything in the filename. But your way is fine, too. It's up to you, of course.

File: docs/standard/linq/toc.yml (hunk 2 above)
Comment (jameshkramer) [reply docs/standard/linq/toc.yml]: That's fine. I only made the change because you had made a perform -> do change in a previous PR.

**Review by mairaw — APPROVED** [review]
Thanks @jameshkramer.

**Pull request merged**